app/controllers/lessons_controller.rb

```diff
@@ -27,7 +27,7 @@ class LessonsController < ApplicationController
   before_action :set_course
 
   def index
-    @lessons = @course.lessons.all.where.not(pub_status: "A")
+    @lessons = @course.lessons.all.where.not(pub_status: "D", pub_status: "A")
```

Comment:
I think you can hand an array of options next time, something like `pub_status: ["D", "A"]`